### ISSUE TITLE
Feedback on "A Guide to TF Layers: Building a Convolutional Neural Network" tutorial

### ISSUE THREAD

**Issue by Vayel** [tensorflow/tensorflow#8301]
This [tutorial](https://www.tensorflow.org/tutorials/layers) seems to contain some errors:

# Convolutional Layer #1

> The filters argument specifies the number of filters to apply (here, 32)

Are the filters learnt? In that case, you could mention it. Else, how do you choose the types of the filters?

> Our output tensor produced by conv2d() has a shape of [batch_size, 28, 28, 1]: the same width and height dimensions as the input, but now with 32 channels holding the output from each of the filters.

Is not the shape `[batch_size, 28, 28, 32]` instead?

# Pooling Layer #1

> Our output tensor produced by max_pooling2d() (pool1) has a shape of [batch_size, 14, 14, 1]

Isn't it `[batch_size, 14, 14, 32]`, as the pooling does not reduce the number of channels?

> the 2x2 filter reduces width and height by 50%.

Maybe you could say that the total size is reduced by 75% (we keep one pixel out of four).

**Comment by nealwu** [comment]
Hi @Vayel, thanks for reporting.

_Are the filters learnt? In that case, you could mention it. Else, how do you choose the types of the filters?_

Not sure I understand your question. These are typical convolution filters where the weights will be learned via gradient descent during training.

_Is not the shape `[batch_size, 28, 28, 32]` instead?_

Yes. Looks like this is already fixed.

_Isn't it `[batch_size, 14, 14, 32]`, as the pooling does not reduce the number of channels?_

Yep. Just fixed this.

_Maybe you could say that the total size is reduced by 75% (we keep one pixel out of four)._

I changed the text to read "the 2x2 filter reduces width and height by 50% each."

You should see the updates pushed out to the website within 3-4 days.
